internal/handlers/handler_oauth2_jwks.go

```
@@ -3,16 +3,18 @@ package handlers
 import (
 	"encoding/json"
 
+	"github.com/authelia/authelia/v4/internal/oidc"
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(goimports)